src/components/Guilds/Filter/index.tsx

```diff
@@ -24,15 +26,41 @@ const FilterRow = styled.div`
   }
 `;
 
+const ButtonContainer = styled(Flex)`
+  flex-direction: Row;
+`;
+
 export const Filter = () => {
+  //@TODO talk to whoever made this how best to refactor
   const [viewFilter, setViewFilter] = useState(false);
+  const [createProposal, setCreateProposal] = useState(true);
   const { totalFilters } = useFilter();
+
+  const history = useHistory();
+  const location = useLocation();
+  console.log(location);
+
   return (
     <FilterContainer>
       <FilterRow>
         <InputText placeholder="Proposal title" />
-        {isDesktop && <FilterMenu />}
-        {isMobile && (
+        {createProposal && (
+          <ButtonContainer>
+            <Button onClick={() => setCreateProposal(false)}>
+              Proposal state
+            </Button>
+            <Button
+              variant="secondary"
+              onClick={() =>
+                history.push(location.pathname + '/proposalselection')

```

Comment:
Could we rename this to `proposalType`
Sorry, should have included this in issue description